ruby/vampires.rb

```diff
@@ -0,0 +1,62 @@
+#process multiple users
+
+puts "How many employees will be processed?"
+num = gets.chomp.to_i
+counter = 1
+
+#user input
+
+def determine_vampire
+puts "What is your name?"
+name = gets.chomp
+puts "How old are you?"
+age = gets.chomp.to_i 
+puts "What year were you born?"
+dob = gets.chomp.to_i
+puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
+garlic = gets.chomp
+puts "Would you like to enroll in the company health insurance? (y/n)"
+enroll = gets.chomp
+puts "Please list your allergies."
+$allergy = gets.chomp
+
+#determine if vampire
+
+loop do 
+  puts "any other allergies? If not, please enter 'done'"
+  @allergy = gets.chomp
+  break if @allergy == "done"
+  break if @allergy == "sunshine"
+end
+
+if $allergy == "sunshine" || @allergy == "sunshine"
+  puts "Probably a vampire"
+elsif name == "Drake Cula" || name == "Tu Fang"
```

Comment:
Consider the optimum condition when running your code.  What can we do to run only necessary processes?  We know that if the employee has Drac or Fang in their name then they are definitely a vampire.  If that is the case, do we need to run any of the other checks?

Consider:

```
if name_matches_vampire_params
  "Definitely a vampire"
else
  if #other checks
    "Something"
  elsif #more conditions...

  else
  end
end
```